### ISSUE TITLE
Media Monitor not running

### ISSUE THREAD

**Issue by jice-lavocat** [okvic77/docker-airtime#1]
I don't know if your image is still maintained, but I gave it a try.
I was able to start airtime and logged in. However, only once the media monitoring was working. Most of the time it is offline. 

First I traced it back to a bad configuration wirh RabbitMq. Then, after rabbitmq was launched, I got this from `/var/log/airtime/media-monitor/media-monitor.log` :  

`2016-02-10 17:50:12,844 INFO - [MainThread] [launcher.py : setup_global()] : LINE 110 - Attempting to set the locale...
2016-02-10 17:50:12,855 INFO - [Thread-1] [watchersyncer.py : __init__()] : LINE 19 - Created timeout thread...
2016-02-10 17:50:12,855 INFO - [Thread-1] [airtime.py : init_rabbit_mq()] : LINE 35 - Initializing RabbitMQ message consumer...
2016-02-10 17:50:12,870 INFO - [Thread-1] [airtime.py : init_rabbit_mq()] : LINE 47 - Initialized RabbitMQ consumer.
2016-02-10 17:50:12,871 DEBUG - [Thread-1] [api_client.py : __call__()] : LINE 140 - http://localhost:80/api/media-monitor-setup/format/json/api_key/C66QKEUKNJG9A0KYU32I
2016-02-10 17:50:12,872 INFO - [Thread-1] [syncdb.py : reload_directories()] : LINE 49 - <urlopen error [Errno 111] Connection refused>
`

So, 2 questions : do you maintain the image ? If so, do you have any clue on how to fix this ?

**Comment by jice-lavocat** [comment]
I am using docker-compose. 

It seemed to work fine today, after a docker-compose stop/start of the project.
Can't say what changed.

**Comment by okvic77** [comment]
Yes, is a bug. If i remember, the problem was that when you restart one service for streaming the audio (liquidsoap) it wont run again. I'll check with any new version of Airtime.

**Comment by jice-lavocat** [comment]
I don't know why, but now it seems more stable here. So, I can't really reproduce the bug consistently.

**Comment by BlurryFlurry** [comment]
im still getting the same error. i need to find the fix in original repo
